packages/control-property-editor/src/use-theme.ts

```diff
@@ -0,0 +1,19 @@
+import type React from 'react';
+import { useEffect } from 'react';
+import { useLocalStorage } from './use-local-storage';
+
+export type ThemeName = 'dark modern' | 'light modern' | 'high contrast black';
+/**
+ * Use local storage.
```

Comment:
JSDoc seems to be misleading here. It looks like it was copied from `useLocalStorage` function.